src/adapters/interlay.ts

```diff
@@ -55,6 +55,20 @@ export const kintsugiRoutersConfig: Omit<CrossChainRouterConfigs, "from">[] = [
       weightLimit: DEST_WEIGHT,
     },
   },
+  {
+    to: "heiko",
+    token: "KBTC",
+    xcm: {
+      // estimated fee calculated from:
+      // ksm_per_sec = ~233_100_000_000
+      // kint_per_sec = (ksm_per_second * 4) / 3 = 310_800_000_000
+      // xcm_secs = ~(247_189_333 / kint_per_sec) ... calculated using a recent kint transaction
+      // kbtc_per_sec = ksm_per_sec / 1_500_000 ... see https://github.com/interlay/interbtc/blob/18efc1e9194a8591713b9885799a6deb6c0392f2/parachain/runtime/kintsugi/src/xcm_config.rs#L104
+      // estimated kbtc fee = kbtc_per_sec * xcm_secs = ~123.6
+      fee: { token: "KBTC", amount: "200" },
```

Comment:
@sander2 , can you please double check my math here?

Reply:
Confirmed amount is safe in tests. Fees were right around the 123 mark.

Reply:
Wait, given the calculation, I assumed this was for the heiko -> kintsugi direction. But it seems from the naming it's the other direction? Then we can't apply this calculation

Reply:
Interesting. During chopsticks testing, I landed pretty much on the mark just around the 123 mark. The reverse direction (heiko -> kintsugi) was considerably less.

Reply:
Submitted [this commit](https://github.com/interlay/bridge/pull/50/commits/119aad6551ce5e4f4c670c6f0b307f74900ebd9f) to address the fee estimates.